internal/datacoord/compaction_trigger.go

```diff
@@ -813,7 +813,7 @@ func (t *compactionTrigger) ShouldDoSingleCompaction(segment *SegmentInfo, compa
 					zap.Uint64("binlogTimestampTo", l.TimestampTo),
 					zap.Uint64("compactExpireTime", compactTime.expireTime))
 				totalExpiredRows += int(l.GetEntriesNum())
-				totalExpiredSize += l.GetLogSize()
+				totalExpiredSize += l.GetMemorySize()
```

Comment:
For compatible reason, please use LogSize if MemSize is zero, Or previous binlogs may never triggers expiredCompaction

Reply:
Compatibility logic has been done when loading meta.